src/views/Transactions.vue

```diff
@@ -17,9 +17,9 @@
       >
         <v-list-item
           v-if="isRecentLoading"
-          style="position: absolute; top: 20px;"
+          style="position: relative; top: 20px;"
```

Comment:
Should remain absolute. You need to find a parent tag to which you will apply `position: relative`